nautobot/docs/plugins/development.md

```diff
@@ -1023,3 +1024,18 @@ With these three components in place, we can request `/api/plugins/animal-sounds
 
 !!! warning
     This example is provided as a minimal reference implementation only. It does not address authentication, performance, or the myriad of other concerns that plugin authors should have.
+
+## Adding Help Documentation
+
+If you are using the `generic.ObjectEditView` from Nautobot for your object, the form will include a help icon with a link to that object's documentation. In order to resolve this link, the view expects to find the documentation for this object in a specific directory tree:
+
+```no-highlight
+plugin_name/
+  - plugin_name/
+    - static/
+      - plugin_name/
+        - docs/
+          - index.html
+          - models/
+            - object_model.html
```

Comment:
I don't think the first `plugin_name` is actually needed - it's just the folder that contains the plugin root directory, and isn't actually part of the loading path.

```suggestion
plugin_name/
  - static/
    - plugin_name/
      - docs/
        - index.html
        - models/
          - object_model.html
```